backend/src/org/commcare/session/CommCareSession.java

```diff
@@ -192,6 +212,9 @@ public String getNeededData() {
 
```

Comment:
It won't let me comment nearer to the comment, but it would be good to update this method's javadoc to either say "1 of 6" or better yet just remove the numeric portion entirely since it's bound to get stale